server/src/main/scala/uk/gov/homeoffice/drt/crunchsystem/DrtSystemInterface.scala

```diff
@@ -372,6 +378,10 @@ trait DrtSystemInterface extends UserRoleProviderLike
       val splitsCalculator = SplitsCalculator(paxTypeQueueAllocation, airportConfig.terminalPaxSplits, splitAdjustments)
       val manifestCacheLookup = RouteHistoricManifestActor.manifestCacheLookup(airportConfig.portCode, now, system, timeout, ec)
       val manifestCacheStore = RouteHistoricManifestActor.manifestCacheStore(airportConfig.portCode, now, system, timeout, ec)
+
+      if (config.getOptional[Boolean]("feature-flags.populate-historic-pax").getOrElse(false))
```

Comment:
is this flag for not populating locally or preprod ?

Reply:
> is this flag for not populating locally or preprod ?

The flag is so we can populate once in any environment. Once we're happy the data is fully populated we can turn the flag off